examples/vision/zero_dce.py

```diff
@@ -0,0 +1,444 @@
+"""
+Title: Zero-DCE for low-light image enhancement
+Author: [Soumik Rakshit](http://github.com/soumik12345)
+Date created: 2021/09/18
+Last modified: 2021/09/18
+Description: Implementing Zero-Reference Deep Curve Estimation for low-light image enhancement
+"""
+"""
+## Introduction
+
+**Zero-Reference Deep Curve Estimation** or **Zero-DCE** formulates light enhancement
+as a task of image-specific curve estimation with a deep neural network. In this example,
```

Comment:
We need to define curve estimation the first time we mention it, since the reader may not be familiar with it.

Reply:
Included a basic explanation or curve estimation along with relevant references.